src/sentry/api/endpoints/project_details.py

```diff
@@ -103,6 +103,12 @@ class DynamicSamplingSerializer(serializers.Serializer):
     rules = serializers.ListSerializer(child=DynamicSamplingRuleSerializer())
     next_id = serializers.IntegerField(min_value=0, required=False)
 
+    # This negative integer represents the rule id that will be sent by the frontend on every rule creation/update.
+    #
+    # We decided to opt for -1 as UNASSIGNED_ID_VALUE because we decided to reserve 0 for the uniform rule id in order
+    # to avoid making changes in Relay's validation mechanism that supports only positive integers (unsigned integer).
```

Comment:
```suggestion
    # to avoid making changes in Relay's validation mechanism that supports only positive integers (unsigned integers).
```

Reply:
I want to specify the name of the type there but I might as well use the plural to make it more concise. Thanks!